### ISSUE TITLE
Add some helper functions

### ISSUE THREAD

**Issue by drvinceknight** [drvinceknight/nbchkr#56]
Add functionality to write checks.

For example:

```
import nbchkr.check

nbchkr.check.previous_output
nbchkr.check.previous_source_code  # Don't know if this is possible
nbchkr.check.expected_value_of_variable
```

**Comment by drvinceknight** [comment]
Looks like getting contents of previous cell is possible https://stackoverflow.com/questions/27952428/programmatically-get-current-ipython-notebook-cell-output

**Comment by drvinceknight** [comment]
Closed by #58
